pkg/utils/has/controller.go

```diff
@@ -106,17 +112,13 @@ func (h *SuiteController) CreateComponent(applicationName, componentName, namesp
 					GitSource: &appservice.GitSource{
 						URL: gitSourceURL,
 					},
-					ImageSource: &appservice.ImageSource{
-						ContainerImage: containerImageSource,
-					},
 				},
 			},
-			Replicas:   1,
-			TargetPort: 8081,
-			Route:      "",
-			Build: appservice.Build{
-				ContainerImage: outputContainerImage,
-			},
+			Secret:         secret,
+			ContainerImage: containerImage,
```

Comment:
Could we keep the name of the value `containerImageSource` so it's more self-explanatory?